web/app.js

```diff
@@ -151,6 +151,12 @@ var PDFViewerApplication = {
   url: '',
   baseUrl: '',
   externalServices: DefaultExternalServices,
+  /**
+   * Hold Event Listeners bound to an object here to be unbound later
+   * @type {{}}
+   * @private
+   */
```

Comment:
I'm not sure that we need such a verbose comment here (or a comment at all really).